.circleci/config.yml

```diff
@@ -350,7 +350,7 @@ commands:
           name: Install dependencies missing from cache
           command: |
             PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
-            yarn install --frozen-lockfile
+            npm ci
```

Comment:
I saw that code in another PR (@EvanHerman 's one I think), is that intended?

Reply:
@olafleur-godaddy We discussed migrating back to `npm` a few weeks ago in Slack so we remain inline with WordPress core. https://godaddy.slack.com/archives/GKQGZCGK1/p1647969507095719